src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -387,11 +387,73 @@ public void inConditionNonIdField() {
           """
           {
             "find": {
-              "filter" : {"non_id" : {"$in": ["a", "b", "c"]}}
-            }
+                "filter" : {
+                     "username" : {"$in" : ["user1", "user10"]}
+                }
+              }
           }
           """;
+      String expected1 =
+          "{\"_id\":\"doc1\", \"username\":\"user1\", \"active_user\":true, \"date\" : {\"$date\": 1672531200000}}";
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("data.documents", hasSize(1))
```

Comment:
Would be great to have a test that tests we get multiple results when using `$in`. All of these tests assert that there's exactly 1 result, so they don't cover the full functionality of `$in`.

Reply:
thanks, added.